.env.docker.staging

```
@@ -0,0 +1,19 @@
+# Docker Environment Configuration for staging
+# Generated on 2025-09-12T22:47:44.335Z
+
+NODE_ENV=staging
+PORT=3000
+DATABASE_URL=postgres://binaai_user:${DB_PASSWORD}@${DB_HOST}:5432/binaai_staging
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the NODE_ENV key

(UnorderedKey)